src/features/pages/deriv-ez/how-to-start/data.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react'
 import { StaticImage } from 'gatsby-plugin-image'
-import StepperView from 'features/components/atoms/tab/tab-content'
+import StepperContent from 'features/components/atoms/tab/tab-content'
 import Link from 'features/components/atoms/link'
 
 export const deriv_ez_images = {
     step1: (
         <StaticImage
-            src="../../../../images/common/deriv-ez/step-1.png"
-            alt="Deriv GO trading app"
+            src="../../../../images/common/trading-platforms/deriv-ez/step-1.png"
+            alt="trading platform trading app"
             loading="eager"
```

Comment:
Why `loading="eager"` is being used here? Shouldn't it be `lazy`?

Reply:
I'm always refer this message https://deriv-group.slack.com/archives/C02CTVA0LBS/p1677037272787159
Will try to use lazy as well